doc/api/errors.md

```diff
@@ -563,6 +563,17 @@ found [here][online].
   encountered by [`http`][] or [`net`][] -- often a sign that a `socket.end()`
   was not properly called.
 
+<a id="nodejs-error-codes">
```

Comment:
Why don't we let the tools to generate the id for all these? It would be consistent that way.